### PR TITLE
[EAP7-1792]: Provide a quickstart with a remote connection to the broker.

### PR DESCRIPTION
**Pull request by ehsavoie**
Jira: https://issues.redhat.com/browse/EAP7-1792

### CODE REVIEW

**Comment by emmartins** [comment]
QE tested this?

**Comment by emmartins** [comment]
nevermind, this is the AD

File: quickstarts/EAP7-1792_provide_quickstart_for_remote_connection_to_artemis_broker.adoc [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+=== Relevant Installation Types
+// Remove the x next to the relevant field if the feature in question is not relevant
+// to that kind of WildFly installation
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [] Managed domain
+
+* [x] OpenShift s2i
+
+* [x] Bootable jar
```
Comment (zhantemirov) [inline quickstarts/EAP7-1792_provide_quickstart_for_remote_connection_to_artemis_broker.adoc:51]: Is this kind of WildFly installation still relevant?

File: quickstarts/EAP7-1792_provide_quickstart_for_remote_connection_to_artemis_broker.adoc [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Using a remote AMQ broker is a scenario that is becoming more and more used, all the more so as on the cloud this is our recommended configuration.
+We should provide a QuickStart on how to configure EAP to connect to a remote AMQ broker. This QuickStart should also be portable to the cloud.
+It should also be the base of a complete documentation update on how to configure this setup.
```
[the remark this answers is not in the document]
Comment (ehsavoie) [reply quickstarts/EAP7-1792_provide_quickstart_for_remote_connection_to_artemis_broker.adoc:13]: No :)

**Comment by zhantemirov** [comment]
@emmartins LGTM
thanks a lot @ehsavoie

**Review by zhantemirov — APPROVED** [review]

**Comment by ehsavoie** [comment]
Integrated into Wildfly Quickstarts by https://github.com/wildfly/quickstart/pull/649

**Pull request merged**